oshdb/src/main/java/org/heigit/bigspatialdata/oshdb/index/XYGrid.java

```diff
@@ -197,6 +196,62 @@ public long getEstimatedIdCount(final OSHDBBoundingBox data) {
   public int getLevel() {
     return zoom;
   }
+  
+  public static class IdRange implements Comparable<IdRange>, Serializable{
```

Comment:
put it into own source file? it seems to me that this is not specific to the XYGrid class
//edit: same for CellIdRange class below (https://github.com/GIScience/oshdb/pull/79/files#diff-d4b1ff67fed6306c1592fa6fdbbd9692R162)

Reply:
also, tabs -> spaces